src/main/java/seedu/duke/constants/Message.java

```diff
@@ -8,15 +8,20 @@ public enum Message {
     FAREWELL("Thank you for using MoneyMoover! We hope to see you again soon :)"),
     HELP("Here are the commands available:\n"
             + "\t help - show list of commands\n"
-            + "\t add CURRENCY AMOUNT - adds that amount of money into that currency account\n"
+            + "\t add CURRENCY AMOUNT [DESCRIPTION] - adds that amount of money into that currency account\n"
             + "\t balance [CURRENCY] - view balances of accounts\n"
             + "\t\t balance <CURRENCY> - View the balance of the specified currency in the account\n"
             + "\t\t balance - displays all currencies\n"
             + "\t exchange CURRENCY1 CURRENCY2 AMOUNT - transfer funds from a currency1 account\n"
             + "\t\t                                   into its equivalent value in currency2 account\n"
-            + "\t withdraw CURRENCY AMOUNT - withdraws that amount of money from that currency account\n"
+            + "\t withdraw CURRENCY AMOUNT [DESCRIPTION] - withdraws that amount of money from that currency account\n"
             + "\t show-rate CURRENCY1 CURRENCY2 [AMOUNT] - shows the value of each dollar in CURRENCY1 in " +
             "terms of CURRENCY2\n"
+            + "\t trans [FLAG] [SEARCH PARAM]  - Appropriate flags are\n"
+            + "\t                                   - desc : search by the description as search parameter\n"
+            + "\t                                   - c : search by currency as search parameter\n"
+            + "\t                                   - d : search by date as search parameter in the form dd-MM-yyyy\n"
+            + "\t                                   - m : search by month as search parameter in the form MM-yyyy\n"
```

Comment:
This is a duplicated message already performed in line 27, remove this chunk of code.